components/Sidebar/Conversation.tsx

```diff
@@ -63,90 +63,98 @@ export const ConversationComponent: FC<Props> = ({
   }, [isRenaming, isDeleting]);
 
   return (
-    <button
-      className={`flex w-full cursor-pointer items-center gap-3 rounded-lg p-3 text-sm transition-colors duration-200 hover:bg-[#343541]/90 ${
-        loading ? 'disabled:cursor-not-allowed' : ''
-      } ${
-        selectedConversation.id === conversation.id ? 'bg-[#343541]/90' : ''
-      }`}
-      onClick={() => onSelectConversation(conversation)}
-      disabled={loading}
-      draggable="true"
-      onDragStart={(e) => handleDragStart(e, conversation)}
-    >
-      <IconMessage size={16} />
-
+    <div className="relative flex items-center">
       {isRenaming && selectedConversation.id === conversation.id ? (
-        <input
-          className="flex-1 overflow-hidden overflow-ellipsis border-b border-neutral-400 bg-transparent pr-1 text-left text-white outline-none focus:border-neutral-100"
-          type="text"
-          value={renameValue}
-          onChange={(e) => setRenameValue(e.target.value)}
-          onKeyDown={handleEnterDown}
-          autoFocus
-        />
-      ) : (
-        <div className="flex-1 overflow-hidden overflow-ellipsis whitespace-nowrap pr-1 text-left">
-          {conversation.name}
+        <div className="flex w-full items-center gap-3 bg-[#343541]/90 p-3">
+          <IconMessage size={16} />
+          <input
+            className="mr-12 flex-1 overflow-hidden overflow-ellipsis border-neutral-400 bg-[#343541]/90 text-left text-white outline-none focus:border-neutral-100"
```

Comment:
noting one visual UI change here: 
removing the border-bottom here `border-b`. When enabled, the border changes the height of the input, causing the entire layout to shift subtly. removing the border keeps things in place. 

I noted that the "Search Conversations..." also does not have bottom border either, and has a transparent background so I assume we are no worse for doing the same here. If that proves to be an A11y issue, it can be resolved for all text inputs in a later PR.